lib/rules/use-brace-expansion.js

```diff
@@ -3,6 +3,8 @@
 const utils = require('../utils/utils');
 const ember = require('../utils/ember');
 
+const ignoredMacros = ['and', 'or'];
```

Comment:
Instead of blacklisting `and` and `or`, I think we should only apply to `computed()` itself (aka none of the macros)...

Reply:
Should now ignore all computed properties that are member expressions.